### PR TITLE
Create ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-4050](https://cumulocity.atlassian.net/browse/DM-4050)
Original PR: [7580](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7580)

[DM-4050]: https://cumulocity.atlassian.net/browse/DM-4050?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: [UI] Fail firmware update process if device reports unexpected UPDATE_RESULT [GRAFT] (#7500) [GRAFT][release/cd] (#7580)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md]: The Claude AI suggested to use the following title:
```suggestion
title: Firmware update process fails if device reports unexpected UPDATE_RESULT
```

File: content/change-logs/device-management/ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-4050
+version: 1021.4.2
+---
+[UI] Fail firmware update process if device reports unexpected UPDATE_RESULT [GRAFT] (#7500) [GRAFT][release/cd] (#7580)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md]: The Claude AI suggested to use the following body:
```suggestion
In some cases, devices reported an unexpected UPDATE_RESULT status during the firmware update process, but the update was still treated as successful. With this change, the firmware update process will now fail if a device reports an unexpected UPDATE_RESULT status. This prevents incorrectly treating a failed update as successful, ensuring the device firmware is in the expected state after an update.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by JacekGonera — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-4050
+version: 1021.4.2
+---
+Before the change if a device reported an unexpected status or result during a firmware update,
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md]:
```suggestion
Previously, if a device reported an unexpected status or result during a firmware update,
the process simply waited for a valid state. Now, you can configure LWM2M devices to fail the firmware update
process when an unexpected situation is detected.
```

File: content/change-logs/device-management/ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-4050
+version: 1021.4.2
+---
+Previously, if a device reported an unexpected status or result during a firmware update,
+the process simply waited for a valid state. Now, you can configure LWM2M devices to fail the firmware update
+process when an unexpected situation is detected.
+the process would simply wait for a valid state. Now, you can configure LWM2M devices to fail the firmware update
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md]:
```suggestion
```

File: content/change-logs/device-management/ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-4050
+version: 1021.4.2
+---
+Previously, if a device reported an unexpected status or result during a firmware update,
+the process simply waited for a valid state. Now, you can configure LWM2M devices to fail the firmware update
+process when an unexpected situation is detected.
+process when an unexpected situation is detected.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1021-4-2-ui-fail-firmware-update-process-if-device-reports-unexpected-update-result.md:19]:
```suggestion
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**